ngraph/python/tox.ini

```diff
@@ -23,11 +23,8 @@ commands=
   flake8 {posargs:src/ setup.py}
   flake8 --ignore=D100,D101,D102,D103,D104,D105,D107,W503 tests/  # ignore lack of docs in tests
   mypy --config-file=tox.ini {posargs:src/}
-  ; TODO: uncomment the line below when all test are ready (and delete the following line)
-  ; pytest --backend={env:NGRAPH_BACKEND} {posargs:tests/}
-  pytest --backend={env:NGRAPH_BACKEND} tests/test_ngraph/test_core.py
-  pytest --backend={env:NGRAPH_BACKEND} tests/test_onnx/test_onnx_import.py tests/test_onnx/test_ops_binary.py
-
+  pytest --backend={env:NGRAPH_BACKEND} tests/test_ngraph/test_core.py -v
```

Comment:
Is the `test_ngraph` directory coming in another PR?

Reply:
yes